QueryBuilder/Compilers/SnowflakeCompiler.cs

```diff
@@ -14,6 +14,19 @@ public sealed class SnowflakeCompiler : PostgresCompiler
         protected override string LastId =>
             throw new NotSupportedException($"LastId not supported in {EngineCode} compiler");
 
+        public override string CompileAggregateColumn(
+            SqlResult ctx,
+            AggregateColumn column
+        )
+        {
+            if (column.Type == "anyvalue")
+            {
+                column.Alias = column.Alias ?? column.Type;
```

Comment:
We don't really need this line as it already is handled in `base.CompileAggregateColumn`, right?

Reply:
it does now